### PR TITLE
Remember last exported preferences

### PR DESCRIPTION
**Pull request by tschechlovdev**
fixes/implements [#97] (https://github.com/koppor/jabref/issues/97)
Now if the preferences are exported and later import preferences is choosen, then the explorer will start where the preferences are last exported.

### CODE REVIEW

**Comment by oscargus** [comment]
Looks good! Would it make sense to store the location to the preferences so it is consistent between sessions?

**Comment by tobiasdiez** [comment]
Yes, I think @oscargus is right and it make sense to store this in the preferences (we already have something similar in place for import/export of bib files. I think it is called `working directory`). Please also add a changelog entry.

**Comment by tschechlovdev** [comment]
I've stored it now in the preferences and added a changelog entry.

File: src/main/java/net/sf/jabref/gui/preftabs/PreferencesDialog.java [hunk 1]
```diff
@@ -63,6 +63,7 @@
     private final JButton showPreferences = new JButton(Localization.lang("Show preferences"));
 
     private final JButton resetPreferences = new JButton(Localization.lang("Reset preferences"));
+    private Path lastExportedPath;
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/gui/preftabs/PreferencesDialog.java]: not needed?

Comment (tschechlovdev) [inline src/main/java/net/sf/jabref/gui/preftabs/PreferencesDialog.java]: Removed. Travis is failing, but that doesn't seem to be related to this PR.

**Pull request merged**